src/include/miopen/graphapi/opgraph.h

```diff
@@ -0,0 +1,274 @@
+#pragma once
+
+#include <miopen/graphapi/graphapi_tensor.hpp>
```

Comment:
I'm going to rename this file
miopen/graphapi/graphapi_tensor.hpp => miopen/graphapi/tensor.hpp

Reply:
That's fine. It's only one line to update.